ios/Classes/QrReader.swift

```diff
@@ -111,7 +111,7 @@ class QrReader: NSObject {
   let cameraPosition = AVCaptureDevice.Position.back
   let qrCallback: (_:String) -> Void
   
-  init(targetWidth: Int, targetHeight: Int, textureRegistry: FlutterTextureRegistry, options: BarcodeScannerOptions, qrCallback: @escaping (_:String) -> Void) {
```

Comment:
Seeing this error `/flutter_qr_mobile_vision/ios/Classes/QrReader.swift:114:95: error: cannot find type 'VisionBarcodeDetectorOptions' in scope`.

Not a Swift guy, but looks like `BarcodeScannerOptions` is defined in the file, so it's unclear to me how it relates to the permission denied error.

I changed it back to `BarcodeScannerOptions` and did not see error again.